blocks/donation-form/__legend/_for/donation-form__legend_for_method.css

```diff
@@ -0,0 +1,9 @@
+.donation-form__legend_for_method {
+  margin-bottom: 16px;
+}
+
+@media screen and (max-width: 1439px) and (min-width: 768px) {
```

Comment:
observing the accepted practice, using two rules would be more practical

Reply:
I guess it's a better solution than media query with recurring rule, isn't it?